Makefile

```diff
@@ -59,10 +59,14 @@ endef
 
 TEST_TIMEOUT := 20m
 
+# TODO: INTEG_TEST should be functional tests
 INTEG_TEST_ROOT        := ./host
```

Comment:
How about renaming this `./host` dir, too, to something like `./host_tests`? It could have a clearer directory name.

Reply:
Will do in the following PR